erpnext/stock/doctype/packed_item/packed_item.py

```diff
@@ -113,3 +119,9 @@ def get_items_from_product_bundle(args):
 
 def on_doctype_update():
 	frappe.db.add_index("Packed Item", ["item_code", "warehouse"])
+
+def get_old_packed_item_details(old_packed_items):
+	old_packed_items_map = {}
+	for items in old_packed_items:
+		old_packed_items_map[items.item_code] = items.as_dict()
```

Comment:
Key should be based on `(items.item_code, items.parent_item)`, otherwise if an item appears multiple times it will create problem.